public/app/features/scopes/dashboards/ScopesDashboardsService.ts

```diff
@@ -22,10 +22,9 @@ interface ScopesDashboardsServiceState {
 }
 
 export class ScopesDashboardsService extends ScopesServiceBase<ScopesDashboardsServiceState> {
-  static #instance: ScopesDashboardsService | undefined = undefined;
-
-  private constructor() {
+  constructor(private apiClient: ScopesApiClient) {
     super({
+      drawerOpened: false,
```

Comment:
As this is about whether the drawer with the dashboards is opened it seems more appropriate to have it here.

Reply:
Why? For me it seems like ScopesDashboardService is responsible for data fetching

Reply:
The ScopesDashboardsService.ts was actually the only one that interacted with this state prop but it was doing it through `ScopesService.instance?.setDrawerOpened(false);` so in that sense was a sole owner of it anyway.

Also the ScopesSelectoreService.ts even before this refactor had it's own `opened: boolean;` prop for opening it's drawer. So this just makes those 2 services more symetrical.

Maybe at some point it would make sense to move the UI related state props out and into the ScopesService but at this moment this seems like a reasonable step to make the code more clear.